vision/google/cloud/vision/geometry.py

```diff
@@ -73,20 +87,34 @@ def __init__(self, latitude, longitude):
         self._longitude = longitude
 
     @classmethod
-    def from_api_repr(cls, response):
+    def from_api_repr(cls, location_info):
         """Factory: construct location information from Vision API response.
 
-        :type response: dict
-        :param response: Dictionary response of locations.
+        :type location_info: dict
+        :param location_info: Dictionary response of locations.
 
         :rtype: :class:`~google.cloud.vision.geometry.LocationInformation`
         :returns: ``LocationInformation`` with populated latitude and
                   longitude.
         """
-        latitude = response['latLng']['latitude']
-        longitude = response['latLng']['longitude']
+        latitude = location_info['latitude']
+        longitude = location_info['longitude']
         return cls(latitude, longitude)
 
+    @classmethod
+    def from_pb(cls, location_info):
+        """Factory: construct location information from Vision gRPC response.
+
+        :type location_info: :class:`~google.cloud.vision.v1.LocationInfo`
+        :param location_info: gRPC response of ``LocationInfo``.
+
+        :rtype: :class:`~google.cloud.vision.geometry.LocationInformation`
+        :returns: ``LocationInformation`` with populated latitude and
+                  longitude.
+        """
+        return cls(location_info.lat_lng.latitude,
+                   location_info.lat_lng.longitude)
```

Comment:
Also confirmed from the proto:
https://github.com/googleapis/googleapis/blob/7e594903ac603ff3b20976163613123be22d8f1c/google/cloud/vision/v1/image_annotator.proto#L293-L296

Thansk for the link to the `pb2` file, though I find the "true source" (i.e. the proto def) to be easier to parse.